ui/src/store/modules/project.ts

```diff
@@ -84,7 +84,16 @@ const getters: GetterTree<ProjectState, RootState> = {
   },
 
   dimensions (state) {
-    return state.dimensionMetadataCollection?.hasPart || []
+    const dimensions = state.dimensionMetadataCollection?.hasPart || []
+
+    // Sort dimensions with the following priority:
+    //   sh:order / measure dimension / key dimension / property (alphabetically)
```

Comment:
Ah, wasn't the proposal to put measure/key first?

Reply:
If we put measure/key first, users won't have full control over the ordering. But I may be wrong, I did what made sense to me. Did someone write the proposal somewhere?

Reply:
Found it: https://github.com/zazuko/cube-creator/issues/987
I think my solution follows the proposal.

Reply:
👌